Web of Science.js

```diff
@@ -0,0 +1,211 @@
+{
+	"translatorID": "88e11bcb-464d-4b6d-a446-8994e3b865c9",
+	"label": "Web of Science",
```

Comment:
Looks like they've been flipping back and forth. Doesn't help that their domain name doesn't match the title of their website. "Web of Science" is fine then